src/main/java/com/thebois/models/world/structures/House.java

```diff
@@ -0,0 +1,34 @@
+package com.thebois.models.world.structures;
+
+import com.thebois.models.Position;
+
+/**
+ * A structure of type House.
+ */
+public class House extends AbstractStructure {
+
+    /**
+     * Creates a house structure.
```

Comment:
Commenting constructors is difficult imo, but I've had success trying to describe *how* this constructor initializes the object rather than the fact that it does. In this case you could write "Creates a house structure at a given position in the world."